templates/explore/repo_list.tmpl

```diff
@@ -40,7 +40,7 @@
 					</div>
 					<div class="flex-item-trailing">
 						{{if .PrimaryLanguage}}
-							<a class="muted" href="{{$.Link}}?q={{$.Keyword}}&sort={{$.SortType}}&language={{.PrimaryLanguage.Language}}">
+							<a class="muted" href="{{$.Link}}?{{if $.PageIsUserProfile}}tab={{$.TabName}}&{{end}}q={{$.Keyword}}&sort={{$.SortType}}&language={{.PrimaryLanguage.Language}}">
```

Comment:
```suggestion
							<a class="muted" href="{{$.Link}}?tab={{$.TabName}}&q={{$.Keyword}}&sort={{$.SortType}}&language={{.PrimaryLanguage.Language}}">
```

Reply:
If it is not user profile page, we will always have a strange `tab` param in the url.

Reply:
This bug is similar to #26374, a empty `tab` doesn't really matter, but it makes the code a little simpler.

Reply:
I don't think this is right.
e.g. We have no `repositories` tab in the following page.
yourdomain/notifications/watching
![image](https://github.com/go-gitea/gitea/assets/18380374/bd572040-bf38-4942-85ae-bbf4dd11278f)

Reply:
ops, this page can not filter repo by language. We need to fix this in another PR.

Reply:
how can we access `/notifications/subscriptions` from web?
I found this page from source code, but have no idea about how to access it from web.